### PR TITLE
Add path aliases

### PR DESCRIPTION
**Pull request by Ferotiq**
### Description

In this pr I replace all the relative imports with aliases that look nice.

Closes #

### CODE REVIEW

**Comment by Bruception** [comment]
Save exact

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "@typescript-eslint/parser": "^5.11.0",
     "concurrently": "5.3.0",
     "eslint": "8.8.0",
+    "eslint-import-resolver-typescript": "^2.7.0",
```
Comment (Bruception) [inline package.json:32]: Save exacf

**Review by Bruception — CHANGES_REQUESTED** [review]
I think there might be a simpler way to achieve this.

**Comment by Ferotiq** [comment]
I might try this later.

**Pull request closed without merging**